media-gfx/freecad/freecad-1.0_rc2-r1.ebuild

```diff
@@ -294,12 +295,16 @@ src_configure() {
 # configuration. Without those, there are sandbox violation, when it
 # tries to create /var/lib/portage/home/.FreeCAD directory.
 src_test() {
-	pushd "${BUILD_DIR}" > /dev/null || die
-	export FREECAD_USER_HOME="${HOME}"
-	export FREECAD_USER_DATA="${T}"
-	export FREECAD_USER_TEMP="${T}"
-	nonfatal ./bin/FreeCADCmd --run-test 0
-	popd > /dev/null || die
+	(
+		pushd "${BUILD_DIR}" > /dev/null || die
+		export FREECAD_USER_HOME="${HOME}"
+		export FREECAD_USER_DATA="${T}"
+		export FREECAD_USER_TEMP="${T}"
+		nonfatal ./bin/FreeCADCmd --run-test 0
+		ret=${?}
+		popd > /dev/null || die
+		exit "${ret}"
+	) || die
 }
```

Comment:
Why are you making this into a subshell? You can mark these variables as `local -x FREECAD_*` or `local ret` and it will be basically the same thing.